src/components/LoginForm/LoginForm.js

```diff
@@ -1,29 +1,54 @@
-import { Link } from "react-router-dom";
-import { useQuery } from "@apollo/client";
-import { USERS_QUERY } from "../../queries/queries";
-import QueryResult from "../QueryResult/QueryResult";
-import "./LoginForm.css";
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { useLazyQuery } from '@apollo/client';
+import { CREATE_SESSION_QUERY } from '../../queries/queries';
+import ConditionalLink from '../ConditionalLink/ConditionalLink';
+import './LoginForm.css';
 
 function LoginForm({ setUser }) {
-  const { loading, error, data } = useQuery(USERS_QUERY);
+  const [loginValues, setLoginValues] = useState({
+    email: '',
+    password: ''
+  });
+  const [loginUser, { loading, error, data }] = useLazyQuery(CREATE_SESSION_QUERY);
+
+  const handleChange = (event) => {
+    const fieldInput = event.target
+    setLoginValues({ ...loginValues, [fieldInput.name]: fieldInput.value });
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    loginUser({ variables: { email: loginValues.email, password: loginValues.password } });
+    console.log({data})
+    if (data) setUser(data.createSession.user);
+  }
 
   return (
     <section className="page--container column">
-      <QueryResult data={data} error={error} loading={loading}>
-        <p className='welcome'>Welcome to FieldTrippers!</p>
-        <form>
-          <Link to="/trip-type">
-            <button
-              className="primary--button login--button"
-              onClick={() => setUser(data.users[0])}
-            >
-              See App
-            </button>
-          </Link>
-        </form>
-      </QueryResult>
+      <h2>Login Form</h2>
+      <form>
+        <input
+          type="email"
+          name="email"
+          placeholder="Email"
+          onChange={(event) => handleChange(event)}
+        />
+        <input
+          type="password"
+          name="password"
+          placeholder="Password"
+          onChange={(event) => handleChange(event)}
+        />
+        <button 
+          className="primary--button"
+          onClick={(event) => handleSubmit(event)}
+        >
+          Login
+        </button>
+      </form>
```

Comment:
For forms, we want to use the form's `onSubmit` prop instead of the button's `onClick`. This will allow us to use the `enter` key to submit the form :) The `type="submit"` on the button indicates that this specific button should be the one that submits the form, useful if there are other buttons in the form.

```suggestion
      <form onSubmit={(event) => handleSubmit(event)}>
        <input
          type="email"
          name="email"
          placeholder="Email"
          onChange={(event) => handleChange(event)}
        />
        <input
          type="password"
          name="password"
          placeholder="Password"
          onChange={(event) => handleChange(event)}
        />
        <button 
          className="primary--button"
          type="submit"
        >
          Login
        </button>
      </form>
```

Reply:
Thank you! That's now been implemented